src/libraries/Directory.Build.targets

```
@@ -272,4 +278,21 @@
       </ItemGroup>
     </When>
   </Choose>
+
+  <Target Name="LibIntellisenseDocs" Condition="'$(IncludeBuildOutput)' == 'true'">
+   <ItemGroup>
+      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)/1033/$(TargetName).xml" Condition="Exists('$(XmlDocFileRoot)/1033/$(TargetName).xml')">
```

nit: we standardized on `\` slashes.

```suggestion
      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)1033\$(TargetName).xml" Condition="Exists('$(XmlDocFileRoot)1033\$(TargetName).xml')">
```